monitoring/oplog-populator/dashboard.py

```diff
@@ -0,0 +1,362 @@
+    dataSource='${DS_PROMETHEUS}',
+    colorMode='background',
+    noValue='-',
+    reduceCalc='last',
+    targets=[Target(
+        expr='\n'.join([
+            'sum(up{',
+            '    namespace="${namespace}",',
+            '    job="${kafka_connect_job}"',
+            '})',
+        ]),
+    )],
+    thresholds=[
+        Threshold('red', 0, 0.),
+        Threshold('green', 1, 1.),
+    ])
+
+connector_instances = Stat(
+    title='Active Connectors',
+    description='Number of active kafka-connect connector instances',
+    dataSource='${DS_PROMETHEUS}',
+    colorMode='value',
+    noValue='-',
+    reduceCalc='last',
+    targets=[Target(
+        expr='sum(' + Metrics.CONNECTORS.raw() + ')',
+    )],
+    thresholds=[
+        Threshold('red', 0, 0.),
+        Threshold('green', 1, 1.),
+    ])
+
+total_number_buckets = Stat(
+    title='Monitored Buckets',
+    description='Number of buckets configured on the connectors',
+    dataSource='${DS_PROMETHEUS}',
+    colorMode='value',
+    noValue='-',
+    reduceCalc='last',
+    targets=[Target(
+        expr='sum(' + Metrics.BUCKETS.raw() + ')',
+    )],
+    thresholds=[
+        Threshold('green', 0, 0.),
+    ])
+
+max_pipeline_size = Stat(
+    title='Max Pipeline Size',
+    description='Maximum size of the mongo pipeline',
+    dataSource='${DS_PROMETHEUS}',
+    colorMode='value',
+    noValue='-',
+    reduceCalc='last',
+    format=UNITS.BYTES,
+    orientation='horizontal',
+    targets=[Target(
+        expr='max (' + Metrics.PIPELINE_SIZE.raw() + ')',
+    )],
+    thresholds=[
+        Threshold('green', 0, 0.),
+    ])
+
+bandwidth = Stat(
+    title='Bandwidth',
+    description='Bandwidth used for reconfiguring the connectors',
+    dataSource='${DS_PROMETHEUS}',
+    colorMode='value',
+    noValue='-',
+    reduceCalc='last',
+    format=UNITS.BYTES_SEC,
+    orientation='horizontal',
+    targets=[Target(
+        expr='sum (rate(' + Metrics.REQUEST_SIZE() + '))',
+    )],
+    thresholds=[
+        Threshold('green', 0, 0.),
+    ])
+
+config_applied_success = TimeSeries(
+    title='Connector Reconfiguration Rate',
+    dataSource='${DS_PROMETHEUS}',
+    gradientMode='opacity',
+    lineInterpolation='smooth',
+    legendDisplayMode='hidden',
+    fillOpacity=10,
+    spanNulls=True,
+    unit=UNITS.OPS_PER_SEC,
+    targets=[Target(
+        expr='sum (rate(' + Metrics.CONNECTOR_CONFIG_APPLIED('success="true"') + '))',
+    )])
+
+config_applied_failure = TimeSeries(
+    title='Failed Connector Reconfiguration Rate',
+    dataSource='${DS_PROMETHEUS}',
+    gradientMode='opacity',
+    lineInterpolation='smooth',
+    legendDisplayMode='hidden',
+    fillOpacity=10,
+    spanNulls=True,
+    unit=UNITS.OPS_PER_SEC,
+    targets=[Target(
+        expr='sum (rate(' + Metrics.CONNECTOR_CONFIG_APPLIED('success="false"') + '))',
+    )])
+
+
+bucket_distribution = PieChart(
+    title='Bucket Distribution',
```

Comment:
I feel like this is misleading, the Bucket from prometheus and the Bucket from S3... Maybe we can have something easier to understand for a not-so-technical reader?